scripts/route_check.py

```diff
@@ -1,6 +1,39 @@
 #!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 
+"""
+What it is:
+    The routes flow from APPL-DB to ASIC-DB, via orchagent.
+    This tool's job is to verify that all routes added to APPL-DB do
+    get into ASIC-DB.
+
+
+How:
+    NOTE: The flow from APPL-DB to ASIC-DB takes non zero milliseconds.
+    1) Initiate subscribe for ASIC-DB updates.
+    2) Read APPL-DB & ASIC-DB 
+    3) Get the diff.
+    4) If any diff, 
+        4.1) Collect subscribe messages for a second
```

Comment:
should you verify it persist for 3 consecutive reads instead of 1 read?

Reply:
That is another scheme. 3 consecutive reads and look for diff that is consistently same across all to imply that as possible  failures. But this has a hole if a a route was getting removed & added quickly multiple times, then it could be a false positive.

Here we look for SET/DEL in subscription messages to see, if a missed ASIC entry matches with a SET subscribe message, which would have added to ASIC. Similarly look for DEL message for an unaccounted ASIC entry as that would have removed it.